.github/workflows/php.yml

```diff
@@ -29,13 +29,12 @@ jobs:
       fail-fast: false
       matrix:
         php-version:
-          - "7.4"
-          - "8.0"
           - "8.1"
```

Comment:
We are not ready to drop support for 7.4 and 8.0 for Drupal 9.
We are evaluating if we will support 10.x only on 8.1 

We will add 10.x knowing that these tests will fail, till the client SDK is updated to use Symfony 6

```
       include:
          - php-version: "8.1"
            drupal-core: "10.0.x"
          - php-version: "8.1"
            drupal-core: "10.1.x"
        php-version:
          - "7.4"
          - "8.0"
          - "8.1"
        drupal-core:
          # Should update the following as the minimum supported version from Drupal.org
          - "9.4.x"
          - "9.5.x" 
 ```

To make your tests run for 10.x you may have to update the rules module

`composer require 'drupal/rules:3.x-dev@dev'`